games-emulation/melonds/patches/enable_wayland_flag.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/src/frontend/qt_sdl/CMakeLists.txt b/src/frontend/qt_sdl/CMakeLists.txt
```

Comment:
Could you created a patchset here, it can consist of multiple commits also, running with -e after extract will combine the 2 patches into one.

Reply:
Yes I can, but I'm not the author of the "enable_wayland_flag" patch (it was made by an upstream contributor). If a patchset is preferred, I can create one (using the usual commands) and then give proper credits adding a comment in the final file. Please let me know.

Reply:
You can grab that one with `wget https://github.com/melonDS-emu/melonDS/pull/1654.patch` (that includes the auriginal author) :)